modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -241,34 +259,16 @@ private void _assetNotificationQueueEntry(
 			expectedSingleRecipient,
 			notificationRecipientSettingsMap.get("singleRecipient"));
 
-		String actualToRecipient = String.valueOf(
-			notificationRecipientSettingsMap.get("to"));
-
-		String[] actualEmailAddresses = StringUtil.split(actualToRecipient);
+		String[] actualEmailAddresses = StringUtil.split(
+			String.valueOf(notificationRecipientSettingsMap.get("to")));
 
 		String[] expectedEmailAddresses = StringUtil.split(expectedToRecipient);
 
-		Assert.assertEquals(
-			Arrays.toString(actualEmailAddresses),
-			expectedEmailAddresses.length, actualEmailAddresses.length);
-
-		for (String expectedEmailAddress : expectedEmailAddresses) {
-			boolean containsEmailAddress = false;
-
-			for (String actualEmailAddress : actualEmailAddresses) {
-				if (StringUtil.equals(
-						expectedEmailAddress, actualEmailAddress)) {
+		Arrays.sort(actualEmailAddresses);
 
-					containsEmailAddress = true;
+		Arrays.sort(expectedEmailAddresses);
 
-					break;
-				}
-			}
-
-			Assert.assertTrue(
-				actualToRecipient + " does not contain " + expectedToRecipient,
-				containsEmailAddress);
-		}
+		Assert.assertArrayEquals(expectedEmailAddresses, actualEmailAddresses);
```

Comment:
I don't know if the best is:
1. Split twice and after Sort twice OR
2. Split and Sort in sequence. Something like this:

 ```java
String[] actualEmailAddresses = StringUtil.split(
	String.valueOf(notificationRecipientSettingsMap.get("to")));

Arrays.sort(actualEmailAddresses);

String[] expectedEmailAddresses = StringUtil.split(expectedToRecipient);

Arrays.sort(expectedEmailAddresses);

Assert.assertArrayEquals(expectedEmailAddresses, actualEmailAddresses);
```

I see the two approach in the portal :/

Reply:
Yes, I agree that second approach seems easier to read! Thank you